mmdet/models/losses/iou_loss.py

```diff
@@ -11,7 +12,11 @@
 
 
 @weighted_loss
-def iou_loss(pred, target, linear=False, mode='log', eps=1e-6):
+def iou_loss(pred: torch.Tensor,
```

Comment:
@ZwwWayne is that necessary to use `Tensor` here? we can use `from torch import Tensor` at beginning

Reply:
We can do that.